fortanix-vme/eif-tools/src/bin/elf2eif.rs

```diff
@@ -0,0 +1,108 @@
+use std::fs::File;
+use std::io::Write;
+
+use anyhow::Result;
+use clap::{Arg, crate_authors, crate_version};
+use env_logger;
+use log::{debug, info, LevelFilter};
+use nitro_cli::build_from_docker;
+use tempdir::TempDir;
+
+use eif_tools::*;
+
+/// Create a temporary used for creating a docker image.
+fn setup_docker_dir(elf_path: &str) -> Result<TempDir> {
+    const DOCKERFILE: &str = "
+        FROM alpine
```

Comment:
Why `FROM alpine`? Shouldn't this just be an empty docker image other than `/init` (which is the input ELF)?

Reply:
It can indeed be changed to "scratch"